test/controllers/v1/relay_controller_test.exs

```diff
@@ -95,6 +98,72 @@ defmodule Cog.V1.RelayControllerTest do
     assert updated["description"] == @update_attrs.description
   end
 
+  test "relays are enabled in more than just name", %{authed: requestor} do
+    # We create a relay and add a bundle to it so we can query for it in
+    # 'Cog.Relay.Relays'
+    relay = relay("relay-enable-relay-1", "foobar")
+    bundle = bundle("relay-enable-bundle-1")
+    relay_group = relay_group("relay-enable-group-1")
+    add_relay_to_group(relay_group.id, relay.id)
+    assign_bundle_to_group(relay_group.id, bundle.id)
+
+    # We shouldn't see any relays running our bundle yet, because the relay
+    # has not yet announced it's presence.
+    assert Relays.relays_running(bundle.name) == []
+
+    # Relays don't show up as available unless they are online and enabled.
+    # FakeRelay lets us send announcement messages like a real relay, so Cog
+    # will add the relay to the available relays list.
+    FakeRelay.new(relay) |> FakeRelay.get_bundles |> FakeRelay.announce
```

Comment:
Similarly, I wonder if some of these could be coalesced behind a single function call.